### DISCUSSION TITLE
[General][Java SDK (Component)] Add samples in spi-samples

### DISCUSSION THREAD

**Discussion by heliang666s** [apache/dubbo#14861] [General]
### Pre-check

- [X] I am sure that all the content I provide is in English.

### Apache Dubbo Component

Java SPI Extensions (apache/dubbo-spi-extensions)

### Details

 The spi-samples repository is a case library of the extended library to improve the case library cases.In this task, you need to first clarify the usage of SPI (see the official website for details), then compare SPI with SPI-samples, find the ones that exist in SPI but do not exist in the samples, and then add it.**Note: Due to the large number of modules in this project, you can only claim one module at a time, and then come back to claim the second one when you are done.**
Repository link: https://github.com/apache/dubbo-spi-extensions https://github.com/apache/dubbo-spi-samples

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://www.apache.org/foundation/policies/conduct)

**Comment by Similarityoung** [comment]
Hello, I want to try this work about dubbo-rpc-rest, can you assign it to me?

**Reply by heliang666s** [reply]
Welcome to the Dubbo community and start your good first issue~
